src/Parsing/Parser.hs

```diff
@@ -196,6 +184,7 @@ someASTParser TypeScript = Just (SomeASTParser (ASTParser tree_sitter_typescript
 someASTParser TSX        = Just (SomeASTParser (ASTParser tree_sitter_tsx :: Parser (AST [] TSX.Grammar)))
 someASTParser PHP        = Just (SomeASTParser (ASTParser tree_sitter_php :: Parser (AST [] PHP.Grammar)))
 someASTParser Java       = Nothing
+someASTParser Haskell    = Nothing
```

Comment:
I honestly don’t know how this passed CI given that we use `-Werror`.

We _do_ use `-Werror`, right??

Reply:
_Narrator:_ [they didn’t](https://github.com/github/semantic/pull/313/commits/15358057476208588ab0d289a3616785e2e2f933).